clusterloader/README.md

```diff
@@ -0,0 +1,78 @@
+# Cluster Loader
+
+Cluster Loader is a tool we use to deploy large numbers of various Kubernetes objects to a cluster. Currently, Cluster Loader has only been tested against local clusters.
```

Comment:
It doesn't really matter - it just should be an annotation or something, not be embedded in text.